### DISCUSSION TITLE
Unable to import TV Shows

### DISCUSSION THREAD

**Discussion by mihawk90** [Komet/MediaElch#1672] [Q&A]
Still getting used to this and I'm stumbling over the Import function.

So I have some files in an unsorted directory and want to import them, but as soon as I change the Import Type to "TV Show" the Import directory goes empty (and I can't select one either), and consequently the Import button is greyed out. Not sure how to proceed here:
![image](https://github.com/Komet/MediaElch/assets/556162/3e982304-15f8-48a1-a12f-3cbad40d9149)

Directory for TV shows is set up:
![image](https://github.com/Komet/MediaElch/assets/556162/b53fdefe-b05c-4ca6-ba53-852b8e27201f)

Already reloaded directory and restarted ME, no dice... Am I missing something here?

Side note: that bright header is extremely distracting :laughing:

**Comment by mihawk90** [comment]
OK I think I might have just answered my own question...

After adding a Path for TV Shows that already had some (and then Reloading All TV Shows), I'm getting the option to import into an *existing* TV Show. However, there seems to be no option to create a new Show/Directory. This... kind of defeats the purpose for me to even have a Downloads section for importing when I need to import stuff (or at least create the directory) myself to begin with.
Ideally there should be an option "<create new show>" or similar (which would also require selecting which TV Show path to import to if there's multiple set up), that would upon import automatically open the TV Show scraper window (which it already does anyway). After selecting the show it would then create the new directory and continue with the import.

Maybe I'm missing something else or I'm misunderstanding the feature.

On a side note: It seems the "Disabled" field is ignored for Downloads paths

**Reply by bugwelle** [reply]
Hi,

thanks. tbh, the "Import" view is something I never used myself, hence why it's not properly tested (it seems). The original MediaElch author added it. I don't recommend using it, though, as there may very well be other bugs.

Regards,
Andre
